### ISSUE TITLE
How to push files that are in the default .gitignore?

### ISSUE THREAD

**Issue by mvaneijgen** [git-ftp/git-ftp#329]
I have some files that are being compiled on the fly and thus these file are in my `.gitingnore` file, Now when I use `git ftp` I want some folders/files that are in my `.gitingore` to be push when I run the command.

**Comment by adrum** [comment]
[This](https://github.com/git-ftp/git-ftp/blob/master/man/git-ftp.1.md#syncing-untracked-files) might be of some help.

**Comment by mvaneijgen** [comment]
@adrum thanks that helped some what, but as far as I can see now is that I have to do this 

`dist/styles/*.css:assets/styles/layouts/_footer.scss`
`dist/styles/*.css:assets/styles/layouts/_footer.scss`
`dist/styles/*.css:assets/styles/layouts/_header.scss`
`dist/styles/*.css:assets/styles/layouts/_pages.scss`
`...`
`dist/styles/*.css:assets/styles/layouts/_sidebar.scss`

After that I also have to do this for the folders `assets/styles/common` and `assets/styles/compontents` and for all the feature files I will ever create and these are only my styles after that I also have `images`, `fonts` which all can have different file names and or server sub folder deep

Thats seems like no way of working am I missing a really special `regex` snippet that will fixes all my issues or is this the way of going forward?

**Comment by mkllnk** [comment]
No, you are not missing anything. Git-ftp is missing that feature. See also #325.
